### PR TITLE
Added contributing.md

### LINKED ISSUE #9: Move contributing guidelines to a dedicated file
It is a good practice to have a dedicated `contributing.md` where all the contributing guidelines can be found. Move the respective content from `README` to the newly created file.

### PR DESCRIPTION
**Pull request by ivaibhavjindal**
Moved contribution section to a new file Contributing.md from README.md
Closes #9

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Contributing helps people and makes the world simply a better place, without contributors this project would cease to exist.
+
+> Currently the following CSS frameworks are supported:-
+
+* [Bootstrap-3](https://getbootstrap.com/docs/3.3/)
+* [Bootstrap-4](https://getbootstrap.com/)
+* [Materialize css](https://materializecss.com/)
+* [Foudation css](https://foundation.zurb.com/)
+* [Semantic-UI](https://semantic-ui.com/)
+* [Bulma css](https://bulma.io/)
```
Comment (jamesgeorge007) [inline Contributing.md]: Please leave this section as it is.

**Comment by ivaibhavjindal** [comment]
Did the required changes, kindly review.

**Comment by jamesgeorge007** [comment]
@ivaibhavjindal congrats on your very first PR :tada: 
Would you mind addressing #10

**Comment by ivaibhavjindal** [comment]
> @ivaibhavjindal congrats on your very first PR tada
> Would you mind addressing #10

Sure.

**Pull request merged**